tests/features/json.feature

```diff
@@ -1,3 +1,5 @@
+@JSON
```

Comment:
Why?

Reply:
Used locally to only run JSON behat test. Removed it.